app/templates/components/public/speaker-item.hbs

```diff
@@ -1,8 +1,9 @@
 {{#ui-accordion}}
   <div class="title">
     <div class="ui">
-      <img alt="speaker" class="ui medium rounded image" src="{{if speaker.thumbnailImageUrl speaker.thumbnailImageUrl (if speaker.photoUrl speaker.photoUrl '/images/placeholders/avatar.png')}}">
-      <br>
+      <div class="thumbnail">
```

Comment:
class name is thumbnail-square and here, it is thumbnail. Please test that PR is working

Reply:
changes to `thumbnail-square` every where.

Reply:
@iamareebjamal please merge.